.github/workflows/prs.yml

```diff
@@ -0,0 +1,49 @@
+name: CI PR Builds
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: ['2.5', '2.6', '2.7', '3.0']
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true
+
+      - name: 'Run bundle update'
+        run: bundle update
+
+      - name: Run tests
+        run: bundle exec rake spec
+
+      - name: Coveralls Parallel
+        uses: coverallsapp/github-action@master
```

Comment:
The best practice I've seen recommended is to pin a specific version/SHA and use dependabot for upgrading the pinned dependency. Not sure how we want to do it here, though -- this potentially has an impact on the action we generate as well.

Reply:
it's a relatively simple command and maintained by the coveralls devs.  yeah ideally they should have an @v1 branch but alas they do not.    My concern of locking it to a specific hash is if they change something for their infrastructure it'll break anyways..


Anyways..  @ashkulz re-review this branch again though as I finally got "everything" working with GH actions (including all the DB testing used by the other gems)

Next-up is adding in rails 7.0 (since they have RC1 out)